tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,18 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('getTrashedFolder()', function() {
+		it('should make GET request to get trashed folder when called', function() {
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('get').withArgs('/folders/' + FOLDER_ID + '/trash', testParamsWithQs);
+			folders.getTrashedFolder(FOLDER_ID, testQS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
```

Comment:
The test you show here is meant to assert that the BoxClient object has its `get()` method called with the correct arguments when `folders.get()` is called.  The tests you have for `getTrashedFolder()` both reference the `defaultResponseHandler()` method, which is not actually used in your method.